README.md

```diff
@@ -527,38 +570,137 @@ respect cancellation, the shutdown may never complete.
 The Python SDK is built to work with Python 3.7 and newer. It is built using
 [SDK Core](https://github.com/temporalio/sdk-core/) which is written in Rust.
 
-### Local development environment
+### Building
+
+#### Prepare
+
+To build the SDK from source for use as a dependency, the following prerequisites are required:
+
+* [Python](https://www.python.org/) >= 3.7
+* [Rust](https://www.rust-lang.org/)
+* [pipx](https://github.com/pypa/pipx#install-pipx) (only needed for installing the two dependencies below)
```

Comment:
```suggestion
* [pipx](https://github.com/pypa/pipx#install-pipx) (recommended method for installing the two dependencies below)
```

Reply:
I am actually gonna remove `pipx`, we don't need it.